src/sc_kpm/common/sc_module.py

```diff
@@ -35,13 +36,16 @@ class ScModule(ScModuleAbstract):
     _agents: List[ScAgentAbstract] = []
 
     def add_agent(
-        self, agent: Type[ScAgentAbstract], element: Union[str, ScAddr], event_type: ScEventType
+        self,
+        agent: Type[ScAgentAbstract],
+        element: Union[str, ScAddr] = QuestionStatus.QUESTION_INITIATED.value,
+        event_type: ScEventType = ScEventType.ADD_OUTGOING_EDGE,
     ) -> ScModuleAbstract:
         self._reg_params.append(RegisterParams(agent, element, event_type))
         return self
```

Comment:
Add one more level of tabs, parameters tab level should be not equal to the function tab level